teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher
+  def initialize(id, age, specialization, name = 'Unknown')
+    super(id, age, name)
```

Comment:
- [ ]  [Optional] If you add the recommendation above, you can delete `id` from lines 4 and 5.

- [ ] [Required] Please add `parent_permission` to the parameters and add a default value of `true` because the `teacher` class is supposed to inherit `parent_permission` too from the person class.